cmd/upgrade/accountroles/cmd.go

```diff
@@ -242,10 +242,8 @@ func run(cmd *cobra.Command, _ []string) {
 	case aws.ModeAuto:
 		if isUpgradeNeedForAccountRolePolicies {
 			reporter.Infof("Starting to upgrade the policies")
-			err = r.WithSpinner(func() error {
```

Comment:
Our AWS SDK v2 rebase incorrectly bought this back in, so just returning us to previous state.